entrypoint.sh

```diff
@@ -7,227 +7,277 @@ set -o pipefail
 
```

Comment:
I have no explanation for why this happened.  Note the # of scanned files between the two.  I blame the `.git` folder for black magic:

```bash
$ docker run -it -v $(pwd):/goat seiso/goat:c230b42

Running Seiso Linter
--------------------------

Running linter: ruff
Running linter: hadolint
Running linter: kubeconform
Running linter: dockerfile_lint
Running linter: markdown-link-check
Running linter: cspell
Running linter: cfn-lint
Running linter: jscpd
Running linter: actionlint
Running linter: markdownlint
Running linter: textlint
Running linter: black
Running linter: mypy
Running linter: shfmt
Running linter: shellcheck
Running linter: yamllint

Scanned 277 files
Excluded 0 files

INFO:  shellcheck completed successfully
INFO:  hadolint completed successfully
INFO:  ruff completed successfully
INFO:  kubeconform completed successfully
INFO:  jscpd completed successfully
INFO:  yamllint completed successfully
INFO:  shfmt completed successfully
INFO:  actionlint completed successfully
INFO:  black completed successfully
INFO:  cspell completed successfully
INFO:  dockerfile_lint completed successfully
INFO:  markdownlint completed successfully
INFO:  mypy completed successfully
INFO:  markdown-link-check completed successfully
INFO:  cfn-lint completed successfully
INFO:  textlint completed successfully
INFO:  Linters found no errors.
```

```bash

$ docker run -it -v $(pwd):/goat seiso/goat:c230b42

Running Seiso Linter
--------------------------

Running linter: ruff
Running linter: hadolint
Running linter: kubeconform
Running linter: dockerfile_lint
Running linter: markdown-link-check
Running linter: cspell
Running linter: cfn-lint
Running linter: jscpd
Running linter: actionlint
Running linter: markdownlint
Running linter: textlint
Running linter: black
Running linter: mypy
Running linter: shfmt
Running linter: shellcheck
Running linter: yamllint

Scanned 482 files
Excluded 0 files

INFO:  jscpd completed successfully
INFO:  hadolint completed successfully
INFO:  ruff completed successfully
INFO:  markdown-link-check completed successfully
INFO:  yamllint completed successfully
INFO:  actionlint completed successfully
INFO:  textlint completed successfully
INFO:  cspell completed successfully
INFO:  black completed successfully
INFO:  mypy completed successfully
INFO:  shellcheck completed successfully
INFO:  kubeconform completed successfully
INFO:  shfmt completed successfully
INFO:  markdownlint completed successfully
INFO:  dockerfile_lint completed successfully
INFO:  cfn-lint completed successfully
INFO:  Linters found no errors.
```

```bash
$ find . | wc -l
    1267
```